### ISSUE TITLE
Support rxjs 6

### ISSUE THREAD

**Issue by marcinwadon** [gcanti/fp-ts-rxjs#6]
Hey @gcanti 
I think lib should be updated to use the newest version of `fp-ts`, `ts` and `rxjs`. What do you think?
I can try to do that but it will take me some time as I'm newbie in `fp-ts`

**Comment by marcinwadon** [comment]
@gcanti Hey, I have a problem with declaration merging because after that:
```
declare module 'rxjs' {
  interface Observable<T> {
    _URI: URI
    _A: T
  }
}
```
that `Observable` is considered as the only one (seems like the original one is forgotten). Without that type inference does not work properly.

Here you can find the current code: https://github.com/marcinwadon/fp-ts-rxjs/blob/feature/rxjs/src/Observable.ts

**Comment by gcanti** [comment]
When using module augmentation you must augment the original definition file, try this

```ts
declare module 'rxjs/internal/Observable' {
  interface Observable<T> {
    _URI: URI
    _A: T
  }
}
```

**Comment by marcinwadon** [comment]
Thanks @gcanti, fair point

**Comment by gcanti** [comment]
Released `fp-ts-rxjs@0.5.0`
